Updated for new formats, made into Gem with bin/rawk_log, fixed some bugs

ianheggie opened the pull request:
Updated for new formats, made into Gem with bin/rawk_log, fixed some bugs

Includes renaming rawk_log to avoid conflict with an existing gem called rawk (related to awk).

Tested patch with rails 1.2, 2.3, 3.2 and log format listed for rails 4.0

Includes tests with travis-ci

Also more resilient to corrupted log entries.

yoon commented:
Ian, thanks for the contributions. I noticed the build has an error on CI on 1.8.7. Are you dropping support?

ianheggie commented:
```
hi,
I will have a look at that error, as I am not dropping 1.8.7 support
(I have two projects still using that version)
IanOn 08/01/14 13:57, yoon wrote:

  Ian, thanks for the contributions. I noticed the build has an
    error on CI on 1.8.7. Are you dropping support?
  —
    Reply to this email directly or view
      it on GitHub.

-- 
```

Regards,
Ian Heggie

Heggie Enterprises Pty Ltd
email:  ian@heggie.biz
mobile: 0425 856 328
skype:  ian.heggie or (03) 9005 6328

ianheggie commented:
```
Hi yoon,
no, I am not dropping support for 1.8.7, because two of my projects
still use that.
I have also added some extra tests and fixed a bug (it was only
adding (pid: N) on the last line when a multi line message was being
logged, eg like rails 2.3.18 uses)
gem version 2.4.2
IanOn 08/01/14 13:57, yoon wrote:

  Ian, thanks for the contributions. I noticed the build has an
    error on CI on 1.8.7. Are you dropping support?
  —
    Reply to this email directly or view
      it on GitHub.

-- 
```

Regards,
Ian Heggie

Heggie Enterprises Pty Ltd
email:  ian@heggie.biz
mobile: 0425 856 328
skype:  ian.heggie or (03) 9005 6328
